### PR TITLE
Change tests to expect list, not generator

### PR DESCRIPTION
**Pull request by scrambldchannel**
I think changes in https://github.com/cubewise-code/tm1py/pull/325 need the tests to be re-worked as some are now failing for me. I've made a few changes to get them working again. I think they need a bit more a closer look but maybe this will do in the meantime? 

https://github.com/cubewise-code/tm1py/issues/301

### CODE REVIEW

File: Tests/Cell.py [hunk 1]
```diff
@@ -750,11 +761,9 @@ def test_execute_mdx_values(self):
         cell_values = self.tm1.cubes.cells.execute_mdx_values(mdx)
         self.assertIsInstance(
             cell_values,
-            types.GeneratorType)
+            list)
```
Comment (MariusWirtz) [inline Tests/Cell.py:764]: thanks for the catch 👍

File: Tests/Cell.py [hunk 2]
```diff
@@ -1743,6 +1751,7 @@ def test_clear_with_mdx_happy_case(self):
 
         self.tm1.cells.clear_with_mdx(cube=CUBE_NAME, mdx=mdx)
 
+        # this may need looking at
         value = next(self.tm1.cells.execute_mdx_values(mdx=mdx))
```
Comment (MariusWirtz) [inline Tests/Cell.py:1755]: I suppose here we would need to write
``` python
value = self.tm1.cells.execute_mdx_values(mdx=mdx)[0]
```
Don't you think?

Comment (scrambldchannel) [reply Tests/Cell.py:1755]: Yeah, that should work, I left it alone because I didn't have an 11.7 server to test it with.

Comment (MariusWirtz) [reply Tests/Cell.py:1755]: I see. I will accept the MR and take of these with my local 11.7 instance.

**Pull request merged**